BibBuddy/app/src/main/res/values/strings.xml

```diff
@@ -64,10 +64,8 @@
     <string name="deleted_shelves">Ok, wir haben die Regale gelöscht.</string>
 
     <!-- Library | Help -->
-    <string name="library_help">Um ein neues Regal anzulegen, drücken Sie den <b>+-Button</b>. Danach
-        geben Sie einen Namen ein. In dem Regal können Sie anschließend Ihre Bücher speichern und zu
-        diesen Notizen schreiben. Für weitere Informationen öffnen Sie das Menü rechts oben und
-        klicken Sie auf <b>Hilfe</b>. </string>
+    <string name="library_help">Mit dem <b>+-Button</b> kannst du ein neues Regal anlegen, in dem du
+        Bücher und Notizen speichern kannst! Mehr Infos gibt\'s rechts oben unter &quot;Hilfe&quot;.</string>
```

Comment:
The quotes are not shown in the empty views.

Reply:
Did for all!